mjcf_to_sdformat/mjcf_to_sdformat/converters/model.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
I think this file should be named `world.py` since it converts MJCF to an 
SDFormat World. We can refactor the model creation part into `model.py` so that 
we can add the option to convert MJCF files to models later on.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/21b76c3da90666223352811af70d216f8f722e68